src/Nancy/GlobalizationConfigurationExtensions.cs

```diff
@@ -13,10 +13,12 @@ public static class GlobalizationConfigurationExtensions
         /// </summary>
         /// <param name="environment">An <see cref="INancyEnvironment"/> that should be configured.</param>
         /// <param name="supportedCultureNames">Cultures that the application can accept</param>
-        public static void Cultures(this INancyEnvironment environment, IEnumerable<string> supportedCultureNames)
+        /// <param name="defaultCulture">Used to set a default culture for the application <remarks>If not specified the first supported culture is used</remarks></param>
+        public static void Cultures(this INancyEnvironment environment, IEnumerable<string> supportedCultureNames, string defaultCulture = null)
         {
             environment.AddValue(new GlobalizationConfiguration(

```

Comment:
If no `defaultCulture` is provided, then we should _attempt_ to use `supportedCultureNames.First()`. Emphasis on _attempt_ because we still need to be sure we don't get a `NullRefererenceException` and so on. 

1. There is a remark that seem to suggest this is the case but I can't see the code that actually performs this task
2. The remark should not be inside the `<param name=..` but rather on the line below